src/confcom/azext_confcom/template_util.py

```diff
@@ -147,10 +178,16 @@ def process_env_vars_from_template(image_properties: dict) -> List[Dict[str, str
                     f"Environment variable with value: {value} is missing a name"
                 )
 
-            if value:
-                if config.ACI_FIELD_TEMPLATE_PARAMETERS in value:
-                    response = input(f'Create a wildcard policy for the environment variable {name} (y/n): ')
-                    if response.lower() == 'y':
+            if value is not None:
+                param_check = find_value_in_params_and_vars(
+                    params, vars_dict, value, ignore_undefined_parameters=True)
+                pattern = parameter_and_variable_regex
+                param_name = re.findall(pattern, value)
```

Comment:
Rather than set a local variable equal to the global constant, I think it makes it easier to find the regex value if you just plug it into the function directly.